src/schema/linkml/bertron_schema.yaml

```diff
@@ -108,6 +108,36 @@ classes:
         description: Permanent resolvable URI for the entity at the data source.
         range: uriorcurie
         required: true
+      properties: 
+        description: Provide metadata that may be relevant and applicable to the entity you're describing.
+        multivalued: true
+        required: false ## I think at least 1 metadata attribute should be required?
+        range: string
+        examples:
+          - value_object:
+            - attribute: 
+                id: MIXS:nnnnn
+                label: total phoshorous
+              has_raw_value: "2.2 ppm"
+              has_unit: ppm
+              has_numeric_value: 2.2
```

Comment:
ATM, the schema validator won't validate arrays or objects with nesting; the simplest fix will be to remove these examples from the schema for now and just keep them in the data directory. I will see if there's something that can be done to fix up the validator.

Reply:
@cmungall thoughts?

I'm good with just mocing them into a different directory for now until there's a fix.